tests/test_ops.py

```diff
@@ -1734,7 +1743,12 @@ def test_mathtransform():
         # {"opName": "reduce_mean", "outName": "ragged/reduce_mean/2d_a1", "varShapes":[[]], "varTypes":["float32"], "varInit":["ragged2d"], "axis":1, "keepdims":False},
         # {"opName": "sqrt", "outName": "ragged/sqrt/2d", "varShapes":[[]], "varTypes":["float32"], "varInit":["ragged2d"]},
 
-        {"opName": "strings_split", "outName": "ragged/sqrt/2d", "varShapes":[[]], "varTypes":["string"], "varInit":["string2"], "split":" "},
+        # {"opName": "strings_split", "outName": "ragged/sqrt/2d", "varShapes":[[]], "varTypes":["string"], "varInit":["string2"], "split":" "},
+        # {"opName": "bitcast", "outName": "bitcast", "varShapes":[[1,10]], "varTypes":["float32"], "varInit":["uniform", "uniform"], "dtype":tf.float32},
+        # {"opName": "bitwise_or", "outName": "bitwise_or", "varShapes":[[1,10], [1,10]], "varTypes":["int64", "int64"], "varInit":["uniform_int10", "uniform_int10"]},
+        # {"opName": "bitwise_xor", "outName": "bitwise_xor", "varShapes":[[1,10], [1,10]], "varTypes":["int64", "int64"], "varInit":["uniform_int10", "uniform_int10"]},
```

Comment:
Bitwise and/or: again, proper coverage.
The ops supports int8-64 and uint8-uint64.
We don't necessarily need test cases for every single datatype there, but we definitely need more than one.
When generating different cases, also vary the shapes. For example an int8 scalar, uint16 rank 2, etc etc

Reply:
Fixed.